File: dom/nodes/Document-contentType/support/contenttype_setter.php
<?php
$type = $_GET["type"];
$subtype = $_GET["subtype"];
if(!empty($type) && !empty($subtype)) {
  header("Content-Type: " . $type . "/" . $subtype);
}

$removeContentType = $_GET['removeContentType'];
if(!empty($removeContentType)) {
  header_remove("Content-Type");
}

$mimeHead = $_GET["mime"];
if(!empty($mimeHead)) {
  echo "<meta http-equiv=\"Content-Type\" content=\"" . $mimeHead . "; charset=utf-8\"/>";
}

File: dom/nodes/Document-contentType/support/contenttype_setter.py
def main(request, response):
    type = request.GET.first("type", None)
    subtype = request.GET.first("subtype", None)
    if type and subtype:
        response.headers["Content-Type"] = type + "/" + subtype

    removeContentType = request.GET.first("removeContentType", None)
    if removeContentType:
        del response.headers["Content-Type"]

    content = ''
    mimeHead = request.GET.first("mime", None);
    if mimeHead:
        content = '<meta http-equiv="Content-Type" content="%s; charset=utf-8"/>' % mimeHead

    return content
